README.adoc

```diff
@@ -62,6 +62,16 @@ npm install --save-dev @input-output-hk/web-common
 This is necessary if you want to use the CSS, HTML, or JavaScript dependencies
 bundled in this project.
 
+=== How to publish a new release of the library
+After merging the different pull requests that makes a release you need to figure the next version number of the library using semantic versioning. In the main branch, an administrator of this repository needs to:
+
+1. Update the `web-common` addition in the `package.dhall` to use the next version number.
+2. Commit the changes
+3. Run `npm version <major | minor | patch>` which will create a new commit with the version bumped in `package.json` and a new tag for that version
+4. Push the commits and tags with `git push && git push --tags`
+5. Create a new release in GitHub for the new tag. Add `packages.dhall` to the new release as an attachment
```

Comment:
Updating `packages.dhall` can be not sufficient - for example in `marlowe-playground-client` we require additional pass `update-client-deps`. Maybe we can add here note like:
```
6. Update the `package.dhall` of referencing projects and perform appropriate actions related to such an update if they are required (like running deps updating script, `spago2nix` etc.).
```